data-prep/README.md

```diff
@@ -15,9 +15,59 @@ Data Cleaning and Munging
 
 A number of cleaning steps are necessary to use the raw data.
 
-### Project Filtering
+### Consumption Data Cleaning
+
+**Elec Traces**
+
+* Remove duplicates
```

Comment:
Follow up: since we're not actually coming up with specs based on what we did with hourly data this still needs some work.  For the monthly billing data the fields (`[DIR, SPID, DATE]`) are not present.  What fields in the monthly billing data should we deduplicate on?